examples/panel/stock-market-chatbot/README.md

```diff
@@ -0,0 +1,33 @@
+## LLM-powered Stock Market app with Panel
+
+This application downloads stock market data from Yahoo Finance, populates a duckdb instance, generates a time series plot of the selected stocks. It also allows the user to ask a natural language question about the plot and get a response using the LLM model.
+
+The app is built using Python Yahoo Finance [yfinance](https://pypi.org/project/yfinance/), [Panel](https://panel.holoviz.org/),[DuckDB](https://duckdb.org/), OpenAI's [Vision Model preview API](https://platform.openai.com/docs/guides/vision) and [ImageKit](https://docs.imagekit.io/getting-started/quickstart-guides/python/python_app). The app can be hosted on [Ploomber Cloud](https://www.platform.ploomber.io/).
+
+The app will store the plot generated and save it to ImageKit.io, and then use the OpenAI API to generate a response to the user's question about the plot. 
+
+### Pre-requisites
+
+1. OpenAI API key. Visit their [Documentation](https://platform.openai.com/docs/api-reference/introduction)
+2. ImageKit.io url endpoint, public key, and private key. Visit their [Dashboard](https://imagekit.io/dashboard)
+
+To run the app, you need to install the following packages:
+
+```bash
+conda create -n stock-market-chatbot python=3.10
+conda activate stock-market-chatbot
```

Comment:
no need to add conda commands

Reply:
Resolved https://github.com/ploomber/doc/pull/124/commits/2f2257b1bf6c9ee9c338c18baf68a0fcfc3b5888